python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -143,6 +143,10 @@ def __init__(self,
         self.quantize_available = False
         self.use_amp = False
 
+        # seed setting
+        from pytorch_lightning import seed_everything
+        seed_everything(seed=self.seed)
```

Comment:
it might be a strange behavior if users does not set a seed but get the same result every time they fit the forecaster.

The check for space might be a part of #4987